src/Social/UsersLists/Observers/UsersListsObserver.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Social\UsersLists\Observers;
+
+use Kanvas\Social\UsersLists\Models\UserList;
+
+class UsersListsObserver
+{
+    public function saved(UserList $usersLists): void
+    {
+        $defaultUsersLists = UserList::where('is_default', 1)
+                    ->where('id', '!=', $usersLists->getId())
+                    ->where('users_id', $usersLists->users_id)
+                    ->first();
```

Comment:
@FredPeal just do a count no?

Reply:
No, because if the record exists, I must change the value to 0